### PR TITLE
 Roles and permissions

### PR DESCRIPTION
**Pull request by ArifPMan**
Updates underway

### CODE REVIEW

**Comment by ArifPMan** [comment]
Hey, Numaan,

This is still the first draft as I'm still awaiting screenshots and links.

File: _data/v6_map.json [hunk 1]
```diff
@@ -478,6 +478,18 @@
         "href":"/docs/v6/postman/find_replace",
         "external": false
       },
+      {
+        "title": "Migrating to version7.0",
```
Comment (numaanashraf) [inline _data/v6_map.json]: Let's just call it Postman v7

File: v6/postman/migrating_to_v7.0.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+If you are an admin, you can go to the Postman dashboard and migrate to Postman v7. From your Postman dashboard, click **Settings** and then select **Roles and Permissions** to go to the following screen:
+
+[![migration to v7](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Migrate2.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Migrate2.png)
+
+Click **Migrate my team to v7.0** and Postman takes care of the next steps for you and your team. 
+
+#### Team user on v7
+
+If you are in a team and download Postman version 7.0 when the entire team is on an older version, you will not be able to migrate to version 7.0. Postman displays a message saying you can’t access features on this version of Postman app until the team migrates to version 7.0. If you happen to be on 7.0, click the following links and download your version of Postman:
```
Comment (numaanashraf) [inline v6/postman/migrating_to_v7.0.md]: `your version` -> latest version of Postman v6.

File: v6/postman/migrating_to_v7.0.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+* [Install Postman v6 on macOS 64-bit](https://go.pstmn.io/dl-macos64-v6-latest)
```
Comment (numaanashraf) [inline v6/postman/migrating_to_v7.0.md]: `Download Postman v6 for macOS 64-bit`

File: v6/postman/roles_and_permissions.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
+---
+title: "Roles and permissions"
+page_id: "roles_and_permissions"
+warning: false
+---
+Roles and permissions in Postman have been enhanced to provide a more robust-access control mechanism. In Postman v7.0, you access features through assigned roles that have their own set of permissions and privileges.
```
Comment (numaanashraf) [inline v6/postman/roles_and_permissions.md]: let's not mention `privileges`

**Pull request merged**